src/scripts/main.js

```diff
@@ -2,7 +2,31 @@
 
 const people = require('./lib/people');
 
-// eslint-disable-next-line no-console
-console.log(people); // you can remove it
+const tr = document.querySelector('tr');
 
-// write your code here
+const putLibraryOnTable = () => {
+  for (let i = 0; i < people.length; i++) {
```

Comment:
Replace the traditional for loop with 'people.forEach()' to follow functional programming practices.